pkg/utils/k8sutil.go

```diff
@@ -131,7 +133,11 @@ func GetClient() kubernetes.Interface {
 func BuildOutOfClusterConfig() (*rest.Config, error) {
 	kubeconfigPath := os.Getenv("KUBECONFIG")
 	if kubeconfigPath == "" {
-		kubeconfigPath = os.Getenv("HOME") + "/.kube/config"
+		env := "HOME"
+		if goruntime.GOOS == "windows" {
```

Comment:
I checked Windows Server an the variable "HOME" is valid there. I would rather do a `for{}` loop iterating over the env vars and pick the first one it finds and throw a meaningful error if the configuration is not found.

BTW `%USERPROFILE% == %HOMEDRIVE%%HOMEPATH%`

Reply:
👍